azimuth/utils/ml/perturbation_test.py

```diff
@@ -18,8 +18,7 @@
 
 @dataclass
 class PerturbationTest:
-    """A PerturbationTest is used in PerturbationTestingModule to create different perturbations on the
-    dataset_split."""
+    """Test in PerturbationTestingModule to create different perturbations on the dataset_split."""
```

Comment:
I feel confused reading your new comment. Does this work?
```suggestion
    """Used in PerturbationTestingModule to create different perturbations on the dataset_split."""
```

Reply:
yes!